aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -280,6 +280,11 @@ at::Tensor PackedLinearWeightsQnnp::apply_dynamic_impl(at::Tensor input) {
         (uint8_t*)qnnp_w_data,
         nullptr);
     packB = w.get();
+    // Need to move the check here since we are releasing the weights.
+    TORCH_CHECK(
+        orig_weight.qscheme() == at::kPerTensorAffine,
+        "quantized::linear_dynamic (qnnpack) only supports "
+        "Per Tensor Quantization Scheme");
```

Comment:
I don't get this.  Can the scheme change between when you call prepack and here?  Also, I think I understand "move" in the context of this diff, but I don't think someone looking at the new version of the code in the future will understand what's moving here.

Reply:
This will be removed by a later PR in the stack. I just split the PRs for ease of review and ease of dissecting any future failures. I do understand though it is a cryptic comment.
Regarding qscheme changing. I don't think this is possible. Do you have a reason to believe that it can?